containers/kubernetes/reference-content/kubernetes-cheatsheet.mdx

```diff
@@ -0,0 +1,19 @@
+---
+meta:
+  title: Kubernetes cheatsheet
+  description: A Kubernetes kubectl cheatsheet
+content:
+  h1: Kubernetes cheatsheet
+  paragraph: A Kubernetes kubectl cheatsheet
+tags: kubernetes cheat sheet cheatsheet cheat-sheet
+dates:
+  validation: 2023-03-20
+  posted: 2023-03-20
+categories: 
+  - containers
+  - kubernetes
+---
+
+<Lightbox src="scaleway-kubernetes-cheatsheet.webp" alt="" size="large" />
+
+[Download the cheatsheet](scaleway-kubernetes-cheatsheet.pdf)
```

Comment:
I think we should maintain a standard, and create a page just like the Instances cheatsheet page: https://www.scaleway.com/en/docs/compute/instances/api-cli/cli-cheatsheet/

in the API/CLI category
and including the commands in the page
+ a link to the PDF